cc/mcts_player.cc

```diff
@@ -85,8 +85,9 @@ Coord MctsPlayer::SuggestMove(int num_readouts) {
     TreeSearch(options_.batch_size);
   }
   auto elapsed = absl::Now() - start;
-  std::cerr << "Seconds per 100 reads: " << elapsed * 100 / num_readouts
-            << std::endl;
+  elapsed = elapsed * 100 / num_readouts;
```

Comment:
this isn't measuring elapsed can you update the name to be `timing` or `time_per` or something

Reply:
it is the time elapsed per 100 readouts?